packages/http-server/src/server.ts

```diff
@@ -41,10 +37,11 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
   });
 
   server.addHook('onSend', (request, reply, payload, done) => {
-    if (request.headers.accept === '*/*' && typeof payload !== 'string') {
```

Comment:
@XVincentX what was the reason behind this `*/*`?

Reply:
Let's assume that edge case is covered with thorough tests :]

Reply:
It was, for sure. We're good here.